src/lib/components/ComponentIndex/CardList.svelte

```diff
@@ -1,9 +1,10 @@
 <script>
-	export let title;
+	export let title,
+		id = `category-${encodeURI(title)}`;
```

Comment:
just personal preference, but I'd probably give each its own declaration
```suggestion
	export let title;
	export let id = `category-${encodeURI(title)}`;
```